documentation/deliverables/individual_deliverables/sprint_reporting_dashboard.md

```diff
@@ -0,0 +1,264 @@
+
+**Value:** Establishing robust operational metrics and publishing them to a public-facing dashboard are important steps in creating a data-driven culture that is focused on delivery. Defining clear operational metrics helps align the team internally around key delivery goals and creates a shared understanding around how we measure progress to those goals. Making these metrics publicly available commits us to tracking and monitoring our operational data and encourages us to respond to changes in performance against those metrics. By setting up the appropriate foundation to collect and publish data for delivery metrics, we also make it easier to publish future, more important, metrics about operational and program outcomes.
+
+**Goals**
+
+- Ensure that we are collecting the data needed to calculate proposed metrics
+- Begin to create the infrastructure required for more advanced data analytics
+- Begin to establish a framework to measure project impact and success
+- Build public trust in our approach through transparency around core metrics
+- Improve our ability to understand and manage team capacity and delivery velocity
+
+### User Stories
+<!-- Required -->
+
+- As a **project maintainer**, I want:
+  - to be able to provide input on which metrics we publish and how they are presented in the dashboard, so that I can use the dashboard to meaningfully plan and manage delivery on the project.
+  - to be able to provide additional details about the metrics included in the dashboard, so that other stakeholders know how to interpret these metrics correctly in the context of the project.
+- As an **internal stakeholder**, I want:
+  - the dashboard's key insights to be easy to understand at a glance, so that I don't need to spend a lot of time learning how to interpret the metrics correctly.
+  - to be able to access the dashboard from the HHS network, so that I can easily monitor our metrics when I'm in the office.
+  - to understand how the metrics are calculated, so that I can explain them to other stakeholders.
+  - the data behind the dashboard to be updated regularly, so that I'm not sharing outdated information with other stakeholders.
+- As a **member of the public**, I want:
+  - to be able to easily navigate between the dashboard and other project resources (e.g. Simpler.Grants.gov or GitHub), so that I don't have to bookmark or remember the links for each resource separately.
```

Comment:
Nice, I like this. I think hyperlinks back and forth between resources will be very helpful. It kind of reminds me of https://en.wikipedia.org/wiki/HATEOAS.

Reply:
Oh that's cool, I hadn't heard of HATEOS, but [Odata](https://www.odata.org/getting-started/basic-tutorial/) does something similar with the `@odata.editLink` attribute